jira_automation/README.md

```diff
@@ -0,0 +1,25 @@
+There are three applications in here to facilitate our JIRA/GitHub interaction
+
+1. io.dockstore.jira.ResolutionChecker - used to help find issues open in GitHub that are closed in JIRA. This was to diagnose an issue where 
+Unito was seemingly mysteriously closing JIRA issues at random. It turned out to be because we hadn't properly configured
+a GitHub and JIRA user in Unito -- it's the Unito intended behavior. We currently don't need to run this, although if we have
+a configuration issue again, it could be useful in the future.
+2. io.dockstore.jira.MilestoneChecker - generates GitHub and JQL queries to find mismatches in the JIRA fix version and GitHub milestone. The JIRA fix version is
```

Comment:
i.e. this is still a common problem

Reply:
> i.e. this is still a common problem

The Milestone thing is a common  problem; I typically run it before a backlog grooming meeting and it will usually find a couple of issues out of sync.

Hopefully we won't need the ResolutionChecker again, since we figured out how the cause. But I wanted to keep it in case we onboard somebody incorrectly in the future and it crops up, and we need to go find the issues that were incorrectly closed.

Reply:
(Sorry, comment wasn't a question. Rather, I meant since this is a common problem, this should probably be re-ordered to be first)